interceptors/logging/options.go

```diff
@@ -92,3 +92,7 @@ func DurationToDurationField(duration time.Duration) Fields {
 func durationToMilliseconds(duration time.Duration) float32 {
 	return float32(duration.Nanoseconds()/1000) / 1000
 }
+
+func DefaultRequestDecider(_ string, _ error) bool {
```

Comment:
Is this used?

Reply:
Must have forgotten to remove from the last change